src/spawn.rs

```diff
@@ -171,3 +171,15 @@ pub async fn clear_all<'b, E: sqlx::Executor<'b, Database = Postgres>>(
         .await?;
     Ok(())
 }
+
+/// Check if a job with that ID exists
+pub async fn exists<'b, E: sqlx::Executor<'b, Database = Postgres>>(
+    executor: E,
+    id: &Uuid,
```

Comment:
The ID should just be passed by value here.

Reply:
`Uuid`  implements `Copy` though, are you sure?

Reply:
Oh whoops, I was the one, sorry. Yeah going to change that :D

Reply:
Done. Also cleaned up by using `query_scalar` instead.